lighthouse-core/gather/fetcher.js

```diff
@@ -156,8 +156,8 @@ class Fetcher {
       iframe.src = src;
       iframe.onload = iframe.onerror = () => {
         iframe.remove();
-        delete iframe.onload;
-        delete iframe.onerror;
+        iframe.onload = null;
```

Comment:
come to think of it, why do we do this at all? can `onload` or `onerror` ever refire after it's been fired before and removed from the DOM?

Reply:
it helps me sleep at night

Reply:
also I'm wondering if this holds a reference to itself, so GC would never delete the element ...

Reply:
not since IE8 :)

https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/dd361842(v=vs.85)